LumpNav/LumpNav.py

```diff
@@ -715,12 +710,29 @@ def setupNavigationPanel(self):
 
     self.dual3dButton = qt.QPushButton("Dual 3D")
     self.triple3dButton = qt.QPushButton("Triple 3D")
-
+    
     hbox = qt.QHBoxLayout()
     hbox.addWidget(self.dual3dButton)
     hbox.addWidget(self.triple3dButton)
     self.viewFormLayout.addRow(hbox)
 
+    self.leftFollowCameraButton = qt.QPushButton("Left follow")

```

Comment:
"Left follow" is not a meaningful name for users. For the user the feature name is "auto-center". It would be better to rename variable names accordingly, too.